crates/anvil/core/src/eth/proof.rs

```diff
@@ -5,6 +5,7 @@ use ethers_core::{
     types::{H256, U256},
     utils::rlp,
 };
+use foundry_evm::utils::b256_to_h256;
```

Comment:
Is the plan still to transition the Anvil types to Alloy U256? And we just are going to do for this file and other untouched ones in follow-up?

Reply:
Yep eventually we'll get to them—just letting it happen somewhat naturally by pushing the migration out of the evm crate first.